src/component/feature/form/form-update-feature-container.jsx

```diff
@@ -34,10 +34,13 @@ const prepare = (methods, dispatch) => {
                 delete s.id;
             });
         }
+        delete input.description;
+
         // TODO: should add error handling
         requestUpdateFeatureToggle(input)(dispatch)
             .then(() => methods.clear())
-            .then(() => hashHistory.push(`/features/view/${input.name}`));
+            .then(() => hashHistory.push(`/features`))
+            .then(() => hashHistory.push(`/features/strategies/${input.name}`));
```

Comment:
This feels a bit weird now, change the url to "/features" and then change it back to the current url. Would it not be better to drop both `hashHistory.push()` calls now?

Reply:
But maybe there should be some message saying that the toggle was saved successfully? 

We could for example reuse the `Snackbar` used via the error-store.

Reply:
let me remove the second line which was required when we used the same container for both update and create.